app/Components.scala

```diff
@@ -49,6 +50,7 @@ class AppComponents(context: Context, val config: ApplicationConfiguration)
   val publishingBucket = new EditionsBucket(s3Client, config.aws.publishedEditionsIssuesBucket)
   val previewBucket = new EditionsBucket(s3Client, config.aws.previewEditionsIssuesBucket)
   val editionsPublishing = new EditionsPublishing(publishingBucket, previewBucket, editionsDb)
+  PublishEventsListener.apply(config, editionsDb).start
```

Comment:
Just out of interest -- why the move?

Reply:
`PublishEventsListener` needs an `EditionsDB` now; rather than construct a new one, moving the instantiation here means we can reused the same instance of `EditionsDB` throughout.